src/app/app.component.ts

```diff
@@ -1,17 +1,32 @@
-import { Component } from '@angular/core';
+import {Component, OnDestroy, OnInit} from '@angular/core';
 import {TranslateService} from '@ngx-translate/core';
+import {AuthService} from './service/auth.service';
+import {takeUntil} from 'rxjs/operators';
+import {Subject} from 'rxjs';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+export class AppComponent implements OnInit, OnDestroy {
+
   title = 'ares';
   public isLogged: boolean = true;
```

Comment:
Chyba lepiej inicjalizować tą wartość z `false`

```suggestion
   public isLogged: boolean = false;
```

Tak robisz tutaj
https://github.com/Domnyk/ares/blob/3cea6003f1f75c5027b6c6f797d9f5558519046a/src/app/nav-header/nav-header.component.ts#L11-L14

Reply:
jest inicjalizowana z false. Wcześniej była true (gdy nie było jeszcze logowania)

Reply:
Mi chodzi o to, żeby w `AppComponent` ustawić domyślną wartość `isLogged` na `false`. Wiem, że w momencie kiedy odpali się `ngOnInit()` to wartość ustawi się na odpowiednią ale ktoś kto po raz pierwszy patrzy na ten kod mógłby się zastanawiać czemu jest `isLogged = true`

Reply:
poprawione